src/components/people/People.tsx

```diff
@@ -0,0 +1,120 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonLink } from '../person/Person';
+
+export const People: React.FC = () => {
+  const [listPeople, setListPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    getPeople()
+      .then((res: Person[]) => setListPeople(res))
+      .catch(() => setError(true));
+  }, []);
+
+  const parent = (person:Person) => {
+    return listPeople
+      .filter(
+        (el: Person) => el.name === person.fatherName
+          || el.name === person.fatherName,
+      ).length;
+  };
+
+  const isActive = (name: string) => {
+    return slug.replaceAll('-', ' ').includes(name.toLowerCase());
+  };
+
+  const emptyName = (person:Person) => {
+    return person.motherName ? person.motherName : '-';
```

Comment:
```suggestion
    return person.motherName || '-';
```

Reply:
you are checking just mother?
 i see that you are using it to render empty name for father too